web-ui/src/main/resources/catalog/components/edit/editorhelper/EditorHelperDirective.js

```diff
@@ -137,6 +137,15 @@
 
           // Load the config from the textarea containing the helpers
           scope.config = angular.fromJson($("#" + scope.ref + "_config")[0].value);
+          if (scope.mode !== "suggestion") {
+            scope.config.defaultSelected = {
```

Comment:
One issue is if we use the radio mode the label is displayed

![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/304901ed-6c58-48c3-b9d4-cf18867f9483)


```xml
  <element name="gmd:denominator" id="57.0">
    <label>Denominator</label>
    <description>The number below the line in a vulgar fraction</description>
    <condition>add either a denominator or a distance</condition>
    <helper editorMode="radio_linebreak">
      <option value="5000">1:5´000</option>
```

Reply:
Ok, I'll check. I was unaware of this mode.

Reply:
@fxprunayre I fixed that, but the radio modes doesn't seem to work fine (tested in `main` branch). If you select a value and save the metadata, the radio option selected is lost.

Reply:
Indeed for suggestion of type numbers it was not selecting the value properly. All good.